### PR TITLE
Add: Error Page

### PR DESCRIPTION
**Pull request by T0ffik**

### CODE REVIEW

**Review by wikbro7650 — APPROVED** [review]

**Comment by Veryyapeee** [comment]
Does it look like its being responsive? image

File: src/pages/Error/Error.jsx [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+
 export const ErrorPage = () => {
-    return <div>This is error page</div>;
+    return (
+        <div className='error__page'>
+            <Text customClass='error__text__extraLarge'>Oops!</Text>
+            <Text customClass='error__text__large'>Page Not Found</Text>
+            <Text customClass='error__text__medium'>404</Text>
+            <Text customClass='error__text__small'>
+                The Page You Are Looking For
```
Comment (Veryyapeee) [inline src/pages/Error/Error.jsx:13]: Using BR is generally lame

**Comment by T0ffik** [comment]
> Does it look like its being responsive? image

yeah, my bad I forgot about mobile version

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/pages/Error/Error.jsx (hunk 1 above)
Comment (T0ffik) [reply src/pages/Error/Error.jsx]: Done and it's responsive to

**Review by Veryyapeee — APPROVED** [review]

**Pull request merged**